### ISSUE TITLE
Having the possibility to disable some conversions

### ISSUE THREAD

**Issue by cnicodeme** [klugjo/hexo-autolinker#1]
Hi!

Writing a blog post, I discovered that mentioning "#1" resulted in the text being converted to hashtag URL.

In that case, I don't want it to be converted to an URL, but haven't found a way to disable this specific value.

Is there a way to do so?

**Comment by klugjo** [comment]
Have you tried `hashtag: false` in the config ?

**Comment by cnicodeme** [comment]
I was hoping to keep this one active, but filter with some rule maybe?

**Comment by sdlyfjx** [comment]
Yes! I need this function as well . Because of some URLs in <script></script> is transformed as well
